x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -56,6 +60,15 @@ would be an intersection of API keys permissions and authenticated user's permis
 thereby limiting the access scope for API keys.
 The structure of role descriptor is the same as the request for create role API.
 For more details, see <<security-api-put-role, create or update roles API>>.
++
+--
+NOTE: Due to the way in which this permission intersection is calculated, it is not
+possible to create an API key that is a child of another API key, unless the derived
+key is created without any privileges. In this case, you must explicitly specify a
+role descriptor with no privileges, and the derived API key can be used for
+authentication only, but will not have access to call {es} APIs.
```

Comment:
Technically, it is not "only" the authentication, but a few actions that requires "the same user", e.g. it can also call `GET _security/api_key?owner=true`. Without complicating things too much by explaining all the details, maybe we could just drop the `only` bit.